### ISSUE TITLE
The support structure has a hole on a multiplied piece

### ISSUE THREAD

**Issue by SmartElectronicDesign** [Ultimaker/Cura#18864]
### Cura Version

5.7.0

### Operating System

Wndows 11

### Printer

my personal printer, it is not in the list of printers

### Name abnormal settings

 the support is generated ok only for the first print part on the print bed, the next part is multiplied after the first but the generated support has a hole

### Describe model location

 it doesn't matter where I put the piece

### Describe your model

 yes, the support was ok in the past, with the last two versions of the software I noticed the problem

### Add your .zip here ⬇️

[Testing.zip](https://github.com/Ultimaker/Cura/files/14930759/Testing.zip)

**Comment by HellAholic** [comment]
I do not believe this is a slicing Error as the model is slicing without crash.
I can slice the model without issues using the default profile (0.1) for custom printers so the issue stems from your changes in your profile.
Other 3rd party printers also don't have any issues slicing the model in a one at a time.
![image](https://github.com/Ultimaker/Cura/assets/28710690/759da011-8c1b-48c7-bd21-725c5c9842bd)
I'm changing this ticket to a discussion as it seems you require help recovering from changes you've made to the profile.

**Comment by GregValiant** [comment]
I'm getting the same thing.
If I change the "Support Roof Distance" to 0.1 the problem goes away.

No matter what I tried, "Enable Support Interface" is disabled (grayed out) and non-selectable.  I didn't know that "grayed out" was even possible with QT controls.  Typically a disabled setting is "not visible" rather than grayed out.
